docs/examples/and.rst

```diff
@@ -264,113 +290,174 @@ From NOT to AND: an Important Difference
+:math:`b`, must be set to correlate the qubits strongly, so that in most 
+solutions they have a single value for :math:`z`. (Remember the output in the 
+:ref:`and_solve_by_sampling` section with its lines 5 and 6? The ``chain_b.`` 
+column stands for "chain breaks", which is when qubits in a chain take different 
+values.)
+
+The previous subsection showed that if you mapped the AND problem's variables
+:math:`x_1` and :math:`x_2` to  qubits ``30`` and ``31``, respectively, on the 
+QPU used before, you could not find a single qubit coupled to both for variable 
+:math:`z`. But notice, for example, that qubit ``30`` (representing :math:`x_1`) 
+is coupled to qubit ``45`` and qubit ``31`` (representing :math:`x_2`) is coupled 
+to qubit ``46``. You can represent variable :math:`z` as a chain of qubits 
+``45`` and ``46``, which are coupled:
+
+>>> (45, 46) in sampler.edgelist  # doctest: +SKIP
+True
+>>> sampler_embedded = FixedEmbeddingComposite(sampler, {"x1": [30], "x2": [31], "z": [45, 46]})   # doctest: +SKIP
 >>> sampleset = sampler_embedded.sample_qubo(Q, num_reads=5000,
-...                                          label='SDK Examples - AND Gate')
+...                                          label='SDK Examples - AND Gate') # doctest: +SKIP
 >>> print(sampleset)   # doctest: +SKIP
-  x1 x2  z energy num_oc. chain_.
-0  1  0  0    0.0    2107     0.0
-1  0  1  0    0.0     684     0.0
-2  1  1  1    0.0    1011     0.0
-3  0  0  0    0.0    1193     0.0
-4  1  0  1    1.0       4     0.0
-5  1  1  0    1.0       1     0.0
-['BINARY', 6 rows, 5000 samples, 3 variables]
-
-Optionally, you can use the :doc:`problem-inspector </docs_inspector/sdk_index>`
-to view the solution on the QPU.
-
-.. note:: The next code requires the use of Ocean's problem inspector.
+  x1 x2  z energy num_oc. chain_b.
+0  0  1  0    0.0    1522      0.0
+1  1  1  1    0.0    1219      0.0
+2  0  0  0    0.0    1080      0.0
+3  1  0  0    0.0    1163      0.0
+4  1  0  1    1.0       6      0.0
+5  1  1  0    1.0       6      0.0
+6  0  1  1    1.0       2      0.0
+7  0  1  1    1.0       2 0.333333
+['BINARY', 8 rows, 5000 samples, 3 variables]
+
+The figure below shows this minor embedding of the AND gate onto this 
+Advantage QPU.
+
+.. figure:: ../_images/embedding_pegasus_AND_chain.png
+    :name: Embedding_Pegasus_AND_Chain
+    :alt: image
+    :align: center
+    :scale: 50 %
+
+    An AND gate minor embedded onto an Advantage QPU. Variables :math:`x_1, x_2, z` 
+    (left) are minor-embedded to qubits ``30``, ``31`` and chain ``[45, 46]`` (right). 
+
+
+Chain Strength 
+--------------
+
+For illustrative purposes, the following code purposely weakens the
+:ref:`chain strength <concepts__chain_strength>` (strength of the coupling 
+between qubits ``45`` and ``46``, which represent variable :math:`z`). By 
+explicitly setting chain strength to a low value of 0.25, the two qubits are 
```

Comment:
It would be useful to link to some reference that explains why 0.25 is "low value" (not sure where exactly would that be, though).

Reply:
Added:
![image](https://user-images.githubusercontent.com/34041130/234391086-96f2c84b-ee66-421b-8017-02e4ecc97b71.png)

Reply:
That's great, thanks.